src/api/contenful.mappper.js

```diff
@@ -0,0 +1,18 @@
+export class ContentfulResponseMapper {
+    constructor() {
+        this.posts = []
```

Comment:
Mapper responsibility is just convert response to the data structure you want, contenful api is the place to cache post and check if there is cache before make a api call. Move this state to api class.

Reply:
Done